app.js

```diff
@@ -69,6 +70,7 @@ function initializeForm() {
     form = new Form( formEl, {
         modelStr: modelStr
     }, {
+        'printRelevantOnly': false,
```

Comment:
We'll probably still have to switch this between false/true, depending on what we are working on it. It determines whether skipped (invisible) questions are shown in the print view or not.